<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252525;
color:#f9f9f9;</string>
  </property>
  <widget class="QLabel" name="labelRoundDeletePrepareTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вы уверены, что хотите удалить 
раунд?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCanselDeleteRound">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
   </property>
   <property name="text">
    <string>Cansel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteRound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
border-radius:4;
color:#ffffff;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
